products/bash/1-click-pre-validation.sh

```diff
@@ -100,6 +100,12 @@ if [[ "$missingParams" == "true" ]]; then
   usage
 fi
 
+if [[ $(oc get node -o json | jq -r '.items[].metadata.labels["ibm-cloud.kubernetes.io/zone"]' | uniq | wc -l | xargs) != 1 ]]; then
+  echo "ERROR: 1-click install does not currently support MZR clusters, please try again with nodes from the same region"
+  divider
+  exit 1
+fi
+
```

Comment:
It would be better to follow the other validation examples below and output either an error with a cross or a success with a tick, and on error set `check=1` rather than exit.

Also, I guess this is copied from the above validation of the input parameters, which outputs errors that won't be seen by a user. Note the below errors don't make any mention of "1-click install".

I suggest moving this down after `fi #demoPreparation` on line 188 so it's after check is set to 0, and after the demoPreparation validation (so the output is cleaner and more obvious) and change to:
```suggestion
if [[ $(oc get node -o json | jq -r '.items[].metadata.labels["ibm-cloud.kubernetes.io/zone"]' | uniq | wc -l | xargs) != 1 ]]; then
  echo -e "$cross ERROR: MRZ clusters are not supported, please try again with a cluster with all nodes in a single zone"
  check=1
else
  echo -e "$tick INFO: Cluster nodes are all in a single zone"
fi
```